azuread/resource_application.go

```diff
@@ -316,11 +315,6 @@ func resourceApplicationCreate(d *schema.ResourceData, meta interface{}) error {
 
 	if v, ok := d.GetOk("homepage"); ok {
 		properties.Homepage = p.StringI(v)
-	} else {
-		// continue to automatically set the homepage with the type is not native
-		if appType != "native" {
-			properties.Homepage = p.String(fmt.Sprintf("https://%s", name))
-		}
```

Comment:
this is changing existing behaviour.. should this not wait until 1.0 to change as i will probably break existing configs?

Reply:
Agreed, I will bump to v1.0 👍